### PR TITLE
CRUD functionality for dishes

### PR DESCRIPTION
**Pull request by j0sv**
## PT: https://www.pivotaltracker.com/story/show/150805246 ##

- Added basic functionality for dishes

- Learned how to work with active admin

- Learned that .each is important

### CODE REVIEW

File: features/step_definitions/generic_steps.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Then(/^show me the page$/) do
+  save_and_open_page
+end
+
+When(/^I press "([^"]*)" on "([^"]*)"$/) do |arg1, arg2|
```
Comment (j0sv) [inline features/step_definitions/generic_steps.rb]: This is not used, please remove.

File: features/step_definitions/dishes_steps.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Given(/^the following dishes exist:$/) do |table|
+  table.hashes.each do |dish|
+    FactoryGirl.create(:dish, dish)
+  end
+end
+
+When(/^I press "([^"]*)" for dish "([^"]*)"$/) do |link, dish_name|
+  #pending # Write code here that turns the phrase above into concrete actions
```
Comment (j0sv) [inline features/step_definitions/dishes_steps.rb]: Please remove.

File: features/step_definitions/generic_steps.rb (hunk 1 above)
Comment (faraznaeem) [reply features/step_definitions/generic_steps.rb]: done

File: features/step_definitions/dishes_steps.rb (hunk 2 above)
Comment (faraznaeem) [reply features/step_definitions/dishes_steps.rb]: done

File: spec/factories/dishes.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :dish do
+    name "MyString"
```
Comment (satorinic) [inline spec/factories/dishes.rb]: This is generic name.- Maybe change to real value?

File: spec/factories/dishes.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+    description "MyText"
```
Comment (satorinic) [inline spec/factories/dishes.rb]: Generic name. Change to real value?

Comment (faraznaeem) [reply spec/factories/dishes.rb]: done

File: spec/factories/dishes.rb (hunk 3 above)
Comment (faraznaeem) [reply spec/factories/dishes.rb]: Done

**Review by satorinic — APPROVED** [review]
look good.

**Pull request merged**